index.bs

```diff
@@ -2502,7 +2502,9 @@ Wide Review</h5>
 	and tracking those comments
 	and the [=Working Group=]'s responses,
 	is generally a good practice which would often be considered positive evidence of wide review.
-	[=Working Groups=] <em class="rfc2119">should</em> announce to other W3C Working Groups
+	[=Working Groups=]
+	<em class="rfc2119">should</em> follow the <a href="https://www.w3.org/Guide/process/charter.html#horizontal-review">W3C Horizontal Groups</a>’ review processes,
+	and <em class="rfc2119">should</em> announce to other W3C Working Groups
```

Comment:
Perhaps we can say that Working Groups should follow the [review guidelines](https://www.w3.org/Guide/process/charter.html#horizontal-review) and in particular should enable Horizontal Review by announcing to other W3C Working Groups

Reply:
Isn't that what it says on the lines you highlighted, @dwsinger ? I'm having trouble decoding your comment! Is it just the delta "enable HR by announcing" vs just "should announce"?

Reply:
No, at the moment it references a document about Charters with a link that on text about Horizontal Groups. It's very mixed and could be confusing.

Reply:
Thanks that's clear after you edited the comment.